Cargo.toml

```diff
@@ -36,7 +36,7 @@ edition = "2021"
 
 [workspace.dependencies]
 heck = "0.4.1"
-alloy = { git = "https://github.com/alloy-rs/alloy", rev = "b79db21734cffddc11753fe62ba571565c896f42", default-features = false }
+alloy = { git = "https://github.com/alloy-rs/alloy", rev = "aed6de2c4a0e901c4aa71e4b67c2ed628285d270", default-features = false }
```

Comment:
Could you add a comment why we need this specific commit?

Reply:
Actually no. In some respect, I do not know why this resolves the problem. I thought upgrading to the latest version was the first move to make and it turned out to work that nothing else was needed. I will continue investigating the issue. I am not satisfied with that situation.